audtorch/transforms/functional.py

```diff
@@ -256,7 +256,7 @@ def normalize(signal, *, axis=None):
     return signal / np.maximum(peak, 1e-7)
 
 
-def standardize(signal, axis=None):
+def standardize(signal, axis=None, mean=True, std=True):
```

Comment:
So far, we had `axis` always as the last argument. It doesn't matter, but it makes reading the source code easier as you can directly spot which functions don't provide that option. Also the `*` is missing. So, please change to:
```python
def standardize(signal, *, mean=True, std=True, axis=None):
```

Reply:
Thanks for your changes, could you still please move `axis=None` to the last position.

Reply:
Sorry, I forgot. I moved it in f2d9694